CefSharp/WebBrowserExtensions.cs

```diff
@@ -245,9 +245,10 @@ public static void LoadString(this IWebBrowser browser, string html, string url)
         /// <param name="browser">The ChromiumWebBrowser instance this method extends</param>
         /// <param name="html">The HTML content.</param>
         /// <param name="url">The URL that will be treated as the address of the content.</param>
-        public static void LoadHtml(this IWebBrowser browser, string html, string url)
+        /// <returns>returns false if the Url was not successfully parsed into a Uri</returns>
```

Comment:
Yeah, I noticed that.

Followed https://help.github.com/articles/syncing-a-fork/